apidef/api_definitions.go

```diff
@@ -595,6 +595,8 @@ type APIDefinition struct {
 	EnableDetailedRecording    bool                   `bson:"enable_detailed_recording" json:"enable_detailed_recording"`
 	GraphQL                    GraphQLConfig          `bson:"graphql" json:"graphql"`
 	AnalyticsPlugin            AnalyticsPluginConfig  `bson:"analytics_plugin" json:"analytics_plugin"`
+	EnableSegmentTags          bool                   `bson:"enable_segment_tags" json:"enable_segment_tags"`
+	SegmentTags                []string               `bson:"segment_tags" json:"segment_tags"`
```

Comment:
we already have `tags` here - https://github.com/TykTechnologies/tyk/pull/4030/files#diff-9961ccc89a48d32db5b47ba3006315ef52f6e5007fb4b09f8c5d6d299c669d67R589